docs/entity-editor.rst

```diff
@@ -0,0 +1,63 @@
+######################
+The Entity Editor
+######################
+
+Introduction
+==============
+
+In Bookbrainz, a set of information about a particular author, book, publication, magazine, etc. is referred to as an *Entity*, and the form which we use to add/edit this information is called the **Entity-Editor**.
+We currently have the following types of entities in Bookbrainz:
+
+* Author
+* Work
+* Edition
+* Edition-Group
+* Series 
+* Publisher
+
+
+Different sections of the Entity Editor
+========================================
+
+The Entity-Editor can be broken down into distinct mini-sections which are as follows:
+
+Name Section
+-------------
+Alias Section
+--------------
+Identifier Section
+----------------------
+Entity Section
+------------------
+Relationship Section
+---------------------
+Annotation Section
+---------------------
+Submission Section
+----------------------
```

Comment:
:+1:

If these are going to be filled up afterwards, we can create a table of contents above all these:
```
* :ref:`Name Section`
* :ref:`Alias Section`
etc.
```

This will create a list where each item is a link to the appropriate section

Reply:
Yeah. I was thinking of adding some information about what fields/variables each section deals with. Then we can provide a link to some other part of the documentation where we explain what that particular section is all about (without going into the coding part). Something like `editing.rst` would be a good place for that, I think. There, we can provide information about how to make good and useful edits on Bookbrainz. This would involve explaining all the different sections of the editor.

Reply:
So we have separate documentation for how to use the website and enter edits: https://bookbrainz-user-guide.readthedocs.io/en/latest/
It's pretty empty for the moment being, but that's what you describe: user documentation as opposed to developer documentation

Reply:
> 

I have not completed this section yet. Initially, I wanted to give an outline of what fields a particular section covers, because we are not describing each and every section in the workflow that follows. So yeah, something like this.